### PR TITLE
Add WG I citations

### LINKED ISSUE #2: AR5 WGI
http://www.climatechange2013.org/report/full-report

### PR DESCRIPTION
**Pull request by rgieseke**
From http://www.climatechange2013.org/report/full-report
Closes #2

### CODE REVIEW

**Comment by znicholls** [comment]
> From http://www.climatechange2013.org/report/full-report

Should we include this somewhere? In the file or in a `IPCC-assessment-reports/WGI/source.md` file or something?

**Comment by rgieseke** [comment]
In the Readme probably ...

**Comment by znicholls** [comment]
> In the Readme probably ...

Nice. Let's add that then get it in (maybe a note saying WGIII stuff was typed by hand?)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # citations
-Commonly used citations that are too hard to find
+
+Commonly used citations in BibTeX format that are too hard to find
+
+## AR 5
+
+Only WG I provides BibTeX citations which are included in this repository:
+
+http://www.climatechange2013.org/report/full-report
```
Comment (znicholls) [inline README.md:9]:
```suggestion
http://www.climatechange2013.org/report/full-report

WG III citations have been typed by hand.
```

**Review by znicholls — APPROVED** [review]

**Pull request merged**